plugin/action/mask/mask.go

```diff
@@ -162,19 +171,25 @@ func (p *Plugin) registerPluginMetrics() {
 func (p *Plugin) Stop() {
 }
 
-func appendMask(dst, src []byte, begin, end int) ([]byte, int) {
+func (p *Plugin) appendMask(mask *Mask, dst, src []byte, begin, end int) ([]byte, int) {
 	runeCounter := utf8.RuneCount(src[begin:end])
+	if mask.ReplaceWord != "" {
+		dst = append(dst, []byte(mask.ReplaceWord)...)
+		return dst, len(src[begin:end]) - len(mask.ReplaceWord)
+	}
 	for j := 0; j < runeCounter; j++ {
```

Comment:
`j` usually used in nested cycles. `i` seems better here.

Reply:
do not change this code